src/components/PinterestBase.js

```diff
@@ -1,38 +1,38 @@
 import React from 'react';
 
-import Config from '../util/PinConfig';
-import Const from '../util/PinConst';
+import { Counts } from '../util/PinConfig';
+import { COUNT_TYPES } from '../util/PinConst';
 import i18n from '../util/i18n';
-import Util from '../util/PinUtil';
+import { log } from '../util/PinUtil';
 
 /**
  * @prop {string} (required) type - enum of either 'any' or 'one'
- * 
+ *
  */
 export default class PinterestBase extends React.Component {
 
     constructor(props) {
         super(props)
         this.debounceInitialLogging();
-        i18n.setLang(this.props.lang || 'en');
+        i18n.lang = this.props.lang;
```

Comment:
@zackargyle The answer is "no" ... "because of Jest"
https://facebook.github.io/jest/docs/tutorial-react.html